packages/arb-compiler-evm/arb-test.py

```diff
@@ -499,6 +520,13 @@ def test_tuple(vm):
 # print(vm.code)
 with open("../arb-validator/proofmachine/opcodetesthash.ao", "wb") as f:
     arb.marshall.marshall_vm(vm, f)
+code = arb.compile_block(test_ethhash2)
+vm = arb.compile_program(arb.ast.BlockStatement([]), code)
+vm.static = 4
+print("ethhash2 ", len(vm.code), " codepoints")
```

Comment:
Remove extraneous print statement

Reply:
Current code does this print for every test block.

Reply:
It turns out this mirrors the rest of the file so the change is unneeded.